rasa_sdk/tracing/config.py

```diff
@@ -12,15 +12,35 @@
 from opentelemetry.sdk.trace import TracerProvider
 from opentelemetry.sdk.trace.export import BatchSpanProcessor
 from rasa_sdk.tracing.endpoints import EndpointConfig, read_endpoint_config
+from rasa_sdk.tracing.instrumentation import instrumentation
+from rasa_sdk.executor import ActionExecutor
 
 
-TRACING_SERVICE_NAME = os.environ.get("TRACING_SERVICE_NAME", "rasa_sdk")
+TRACING_SERVICE_NAME = os.environ.get("RASA_SDK_TRACING_SERVICE_NAME", "rasa_sdk")
```

Comment:
Are there other places where we have to update this environment variable? such as docs, helm charts etc?

Reply:
Most likely docs, not sure about helm charts.
I'll confirm that.

Reply:
Nope! Nowhere else.
We may just mention it in the docs or something.